plugins/node/opentelemetry-instrumentation-express/src/types.ts

```diff
@@ -27,7 +27,7 @@ export type ExpressRequestInfo = {
   /**
    * If layerType is undefined, SpanNameHook is being invoked to rename the original root HTTP span.
    */
```

Comment:
I already remove this comment on fork, not sure why GH not picking up that commit 😕

Reply:
There's currently an incident at GitHub, i've seen some changes are not accurately reflected across the Github UI at the moment. Guess we'll have to wait it out :confused: 

https://www.githubstatus.com/